apps/src/templates/levelSummary/FreeResponseAIEvaluation.tsx

```diff
@@ -102,6 +103,9 @@ const FreeResponseAIEvaluation: React.FunctionComponent<
             }
           >
             <div>
+              <FreeResponseAiStudentResponseHeader
+                closeStudentResponses={() => {}}
```

Comment:
This is temporary - right now we have a different PR in the works to open up the free responses.  This will be included in a future PR